File: Client_Root/Client/Assets/Scripts/Network/Messages/ToClient/GameStartToC.cs
using FlatBuffers;

public class GameStartToC : IMessage
{
    public const ushort MESSAGE_ID = MessageID.GameStartToC_ID;

    public float m_fTickInterval;
    public int m_nRandomSeed;
    public int m_nTimeLimit;

    public override ushort GetID()
    {
        return MESSAGE_ID;
    }

    public override IMessage Clone()
    {
        return null; 
    }

    public override byte[] Serialize()
    {
        FBS.GameStartToC.StartGameStartToC(m_Builder);
        FBS.GameStartToC.AddTickInterval(m_Builder, m_fTickInterval);
        FBS.GameStartToC.AddRandomSeed(m_Builder, m_nRandomSeed);
        FBS.GameStartToC.AddTimeLimit(m_Builder, m_nTimeLimit);
        var data = FBS.GameStartToC.EndGameStartToC(m_Builder);

        m_Builder.Finish(data.Value);

        return m_Builder.SizedByteArray();
    }

    public override bool Deserialize(byte[] bytes)
    {
        var buf = new ByteBuffer(bytes);

        var data = FBS.GameStartToC.GetRootAsGameStartToC(buf);

        m_fTickInterval = data.TickInterval;
        m_nRandomSeed = data.RandomSeed;
        m_nTimeLimit = data.TimeLimit;

        return true;
    }
}

File: Server_Root/Server/RoomServer/Messages/ToClient/GameStartToC.cpp
#include "stdafx.h"
#include "GameStartToC.h"
#include "../../../FBSFiles/GameStartToC_generated.h"

GameStartToC::GameStartToC()
{
}

GameStartToC::~GameStartToC()
{
}

unsigned short GameStartToC::GetID()
{
	return MESSAGE_ID;
}

IMessage* GameStartToC::Clone()
{
	return NULL;
}

const char* GameStartToC::Serialize(int* pLength)
{
	FBS::GameStartToCBuilder data_builder(m_Builder);
	data_builder.add_TickInterval(m_fTickInterval);
	data_builder.add_RandomSeed(m_nRandomSeed);
	data_builder.add_TimeLimit(m_nTimeLimit);
	auto data = data_builder.Finish();

	m_Builder.Finish(data);

	*pLength = m_Builder.GetSize();

	return (char*)m_Builder.GetBufferPointer();
}

bool GameStartToC::Deserialize(const char* pChar)
{
	auto data = flatbuffers::GetRoot<FBS::GameStartToC>((const void*)pChar);

	m_fTickInterval = data->TickInterval();
	m_nRandomSeed = data->RandomSeed();
	m_nTimeLimit = data->TimeLimit();

	return true;
}